.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+**If you want your PR addressed quickly, please also reach out to our [support team](https://support.airship.com/)
+so we can understand when you need it reviewed and how it is impacting your use of our services.** We also generally
+will not release new versions of our library without new feature support, a bug fix, or a clear reason from a customer
+why an update is required to minimize how often other customers need to update.
+
+### What does this do and why?
+Please include link to open issue if applicable.
+
+### Additional notes for reviewers
+* If applicable
+
+### Testing
+- [ ] If these changes added new functionality, I tested them against the live API with real auth
+- [ ] I wrote tests covering these changes
+
+* I've tested for Ruby versions:
```

Comment:
this should probably be "node versions" instead

Reply:
Oops! Good catch! Thank you!

Reply:
Changed to Node and added latest LTS node version